package.json

```diff
@@ -56,67 +39,59 @@
   },
   "homepage": "https://github.com/concord-consortium/tectonic-explorer#readme",
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
-    "d3-color": "^1.2.3",
+    "d3-color": "^2.0.0",
     "d3-hsv": "^0.1.0",
-    "d3-interpolate": "^1.3.2",
-    "d3-scale": "^1.0.7",
-    "eventemitter2": "^4.1.0",
-    "firebase": "^5.8.4",
-    "jquery": "^3.4.1",
+    "d3-interpolate": "^2.0.1",
+    "d3-scale": "^3.2.3",
+    "eventemitter2": "^6.4.3",
+    "firebase": "^8.2.7",
+    "jquery": "^3.5.1",
     "kd-tree-javascript": "^1.0.3",
-    "lodash": "^4.17.11",
-    "mobx": "^3.6.2",
-    "mobx-react": "^4.4.3",
+    "lodash": "^4.17.20",
+    "mobx": "^6.1.6",
+    "mobx-react": "^7.1.0",
     "point-in-polygon": "^1.0.1",
-    "postcss": "^6.0.23",
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
+    "postcss": "^8.2.6",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
     "react-toolbox": "2.0.0-beta.13",
-    "react-transition-group": "~2.6.0",
-    "rollbar": "^2.5.2",
-    "screenfull": "^4.0.1",
-    "seedrandom": "^2.4.4",
-    "shutterbug": "^1.1.0",
-    "three": "^0.102.0",
+    "react-transition-group": "~4.4.1",
+    "rollbar": "^2.20.0",
+    "screenfull": "^5.1.0",
+    "seedrandom": "^3.0.5",
+    "shutterbug": "^1.3.2",
+    "three": "^0.119.1",
     "timeseries-analysis": "^1.0.12",
-    "uuid": "^3.3.2"
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/plugin-proposal-class-properties": "^7.3.4",
-    "@babel/plugin-proposal-decorators": "^7.3.0",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/preset-react": "^7.0.0",
-    "@svgr/webpack": "^4.3.0",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.1.0",
-    "babel-loader": "^8.0.5",
-    "copy-webpack-plugin": "^4.6.0",
-    "css-loader": "^2.1.0",
-    "cypress": "^3.3.1",
-    "cypress-image-snapshot": "^3.1.1",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.10.0",
-    "file-loader": "^0.11.2",
-    "fsevents": "^1.2.7",
+    "@svgr/webpack": "^5.5.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.4.1",
```

Comment:
May want to make a note somewhere to update to the official adapter once it's available.